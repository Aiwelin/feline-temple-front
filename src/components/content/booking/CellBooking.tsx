import type { Dayjs } from "dayjs";
import PopConfirmBooking from "./PopConfirmSelectedBooking";

type Props = {
  value: Dayjs;
};

const CellBooking = (props: Props) => {
  return <PopConfirmBooking value={props.value} />;
};

export default CellBooking;
