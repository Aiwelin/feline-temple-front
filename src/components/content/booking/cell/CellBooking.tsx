import type { Dayjs } from "dayjs";
import PopoverSelectBooking from "../popover/PopoverSelectBooking";

type Props = {
  value: Dayjs;
};

const CellBooking = (props: Props) => {
  return <PopoverSelectBooking value={props.value} />;
};

export default CellBooking;
