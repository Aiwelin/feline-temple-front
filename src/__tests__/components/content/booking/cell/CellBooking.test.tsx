import { expect, it } from "vitest";
import CellBooking from "../../../../../components/content/booking/cell/CellBooking";

it("should work as expected", () => {
  const result = <CellBooking />;
  expect(result).toMatchSnapshot();
});
