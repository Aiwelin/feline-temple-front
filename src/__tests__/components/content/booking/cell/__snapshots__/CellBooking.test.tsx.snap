// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`should work as expected 1`] = `<CellBooking />`;
